.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,36 +45,25 @@ Need Support?
 ```
 </details>
 
-### Device (Host) Operating System
-<!-- Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core, etc. -->
-
-### Architecture
-<!-- amd64, arm32, etc. -->
-
-### Container Operating System
-<!-- Linux containers, Windows containers -->
+### Device Information
+* Host OS [e.g. Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core]: 
+* Architecture [e.g. amd64, arm32, arm64]: 
+* Container OS [e.g. Linux containers, Windows containers]: 
 
 ### Runtime Versions
+* iotedged [run `iotedge version`]: 
+* Edge Agent [image tag (i.e. 1.0.0)]: 
+* Edge Hub [image tag (i.e. 1.0.0)]: 
+* Docker/Moby [run `docker version`]: 
 
-#### iotedged
-<!-- Run `iotedge version` -->
-
-#### Edge Agent
-<!-- Image tag (i.e. 1.0.0) -->
-
-#### Edge Hub
-<!-- Image tag (i.e. 1.0.0) -->
-
-#### Docker
-
-```
-<Run `docker version` (`docker -H npipe:////./pipe/iotedge_moby_engine version` for Moby on Windows) and paste here>
-```
+Note: on Windows include the host option `docker -H npipe:////./pipe/iotedge_moby_engine version`
```

Comment:
We've got at least one case where the user was using Linux containers but gave us the Moby version. How about:

"Note: When using Windows containers on Windows, run `docker -H npipe:////./pipe/iotedge_moby_engine version` instead"

?